stl/inc/expected

```diff
@@ -685,6 +689,303 @@ public:
         }
     }
 
+    template <class _Uty = _Err>
+    _NODISCARD constexpr _Err error_or(_Uty&& _Other) const& noexcept(
+        is_nothrow_copy_constructible_v<_Err>&& is_nothrow_convertible_v<_Uty, _Err>) { // strengthened

```

Comment:
We generally (always?) add `strengthened` comment right after `noexcept` (before `{`), like this:
```suggestion
        is_nothrow_copy_constructible_v<_Err>&& is_nothrow_convertible_v<_Uty, _Err>) /* strengthened */ {
```

Reply:
The comment appearing after the brace occurs several times in this header in places I didn't touch. Would you like me to fix all of them?

Reply:
> The comment appearing after the brace occurs several times in this header in places I didn't touch. Would you like me to fix all of them?

(Wow - how did we miss this _13 times_?) Yes, if it's not too much trouble.